limitador/src/storage/redis/redis_cached.rs

```diff
@@ -226,52 +227,44 @@ impl CachedRedisStorage {
         )
         .await?;
 
+        let cached_counters = CountersCacheBuilder::new()
+            .max_cached_counters(max_cached_counters)
+            .max_ttl_cached_counter(ttl_cached_counters)
+            .ttl_ratio_cached_counter(ttl_ratio_cached_counters)
+            .build();
+
+        let cacher = Arc::new(cached_counters);
```

Comment:
`cacher`? Why the rename? what does `cacher` mean?

Reply:
I reckon naming it that way because of the similar composition with the "batcher", but it's true that in this case it doesn't reflect well what it is. I could rename it to `counters_cache`